res/skins/Deere/skin.xml

```diff
@@ -17,8 +17,28 @@
       <attribute persist="true" config_key="[Deere],show_parallel_waveforms">0</attribute>
       <attribute persist="true" config_key="[Microphone],show_microphone">0</attribute>
       <attribute persist="true" config_key="[PreviewDeck],show_previewdeck">0</attribute>
-      <attribute config_key="[Library],show_library">1</attribute>
```

Comment:
Sure, I'll look into it. I am excited to use this together with all the recent changes in master. :D